### ISSUE TITLE
`focus-view next` across all outputs

### ISSUE THREAD

**Issue by sgelb** [riverwm/river#929]
My setup consists of two monitors aka outputs. If my understanding is correct, each of these outputs has its own layout stack. As a result, `focus-view next` will only consider views on the current active output. Is that correct so far?

My question:
How can I build a keymapping that has the same functionality as `focus-view`, but cycles through all views on all outputs?

I cannot get rid of the feeling that I overlook something that is already there, so any pointer in the right direction is appreciated.

**Comment by justinlovinger** [comment]
You could probably write a script that checks if you are at the top or bottom of the stack, focuses the previous or next output if you are, and otherwise focuses the previous or next view. It may not be trivial to implement though.

**Comment by sgelb** [comment]
Yes, that was my idea, too. But I didn't find a way to check where in the stack the currently focused view is, at least not in the current protocols. Or did I miss something?

**Comment by justinlovinger** [comment]
Hmm, ya, that may be missing. That could be a good feature to add.

The stack is global per display, so you will need a command to at least get the lowest and highest stack-number of the current tag-set. Getting all stack-numbers may be more generally useful, like `[3, 5, 9]`. You would also need a command to get the stack-number of the focused view.

**Comment by vide0hanz** [comment]
If I understand correctly, the feature being proposed would wrap focus around to the next output correct? Ie, if there are any views on the next available output and `focus-view-next` is called, then focus on that view.
